indico/modules/events/layout/util.py

```diff
@@ -349,6 +379,43 @@ def get_css_url(event, force_theme=None, for_preview=False):
         return _build_css_url(layout_settings.get(event, 'theme'))
 
 
+def _build_js_url(theme):
+    if ':' in theme:
```

Comment:
```suggestion
    if ':' not in theme:
        return None
```

and then the whole rest doesn't need to be indented in an if/else block anymore